### PR TITLE
Add Question 11 section to master branch

### PR DESCRIPTION
**Pull request by spitalnyd**
The Question 11 section of the Management plan is complete, all checks pass in Travis and a green checkmark is present. This pull request would merge the team4-question11 branch into the master branch. Please comment your suggestions or concerns, feedback would be appreciated.

### CODE REVIEW

File: management.md [hunk 1]
```diff
@@ -35,5 +35,48 @@
...
   software project?
 
-- How will your team know when it is finished with the implementation of the
-  software project?
+  - Finding Defects:
+
+    In order to find the defects in a software project,
+    first check the error
+    messages terminal. Secondly, if there still are
+    effects, check the test
```
Comment (enpuyou) [inline management.md]: I think you mean defects here

File: management.md [hunk 2]
```diff
@@ -35,5 +35,48 @@
...
+    cases and asses their correctness. Lastly,
```
Comment (enpuyou) [inline management.md]: assess?

File: management.md [hunk 3]
```diff
@@ -35,5 +35,48 @@
...
+    check TravisCI to see which
```
Comment (enpuyou) [inline management.md]: There should be a space between "Travis" and "CI". Also, it is worth discussing if you will use `Travis CI` or Travis CI in the future

File: management.md [hunk 4]
```diff
@@ -35,5 +35,48 @@
...
+    would greatly help us communicate and
+    report our progress as well as issues we face.
+
+  - Fix:
+
+    Finding defects is an important part of getting your
+    code to work smoothly. The first step is doing
+    some research to try to find a solution. This
+    research can be found online or in a book. The second step is
+    asking fo help to a classmate, technical
```
Comment (enpuyou) [inline management.md]: This sentence needs to be rephrased. Maybe asking help "from" a classmate?
> asking fo help to a classmate

File: management.md [hunk 5]
```diff
@@ -35,5 +35,48 @@
...
+    leader, and instructor. This could be very useful
+    because they might know your defect in your code.
+    The last step is trial and error with
+    the help of test cases. Sometimes just messing
+    around with your code you can figure out defects.
+    Also, using test cases will help
+    guideline where the error is located.
```
Comment (enpuyou) [inline management.md]: You have mentioned test cases twice here

**Review by enpuyou — CHANGES_REQUESTED** [review]
A good first try! See more comments below. Also, just as a side note, the commit messages in your team are still not up to standard.

**Comment by noorbuchi** [comment]
@enpuyou thanks for the feedback! I've addressed some of the concerns you have with my latest commit, but there are more changes coming soon as we discuss this as a group. Please let me know if you have any additional suggestions. Thank you!

**Review by enpuyou — APPROVED** [review]
Looks good!

**Review by corlettim — APPROVED** [review]

File: management.md [hunk 6]
```diff
@@ -35,5 +35,46 @@
 - What process will the team follow to find, report, and fix defects in the
   software project?
 
-- How will your team know when it is finished with the implementation of the
-  software project?
+  - Finding Defects:
+
+    In order to find the defects in a software project,
+    first check the error
+    messages terminal. Secondly, if there still are
```
Comment (schultzh) [inline management.md]: "error messages in terminal" would be a good way to clarify this point

**Review by schultzh — CHANGES_REQUESTED** [review]
Looks good other than the small change requested below. I will be happy to re-review!

File: management.md (hunk 6 above)
Comment (noorbuchi) [reply management.md]: Thanks for the feedback! I fixed this issue in the latest commit.

**Review by noorbuchi — APPROVED** [review]

**Review by schultzh — APPROVED** [review]

**Pull request merged**